### ISSUE TITLE
Error getting stats: failed to get stats

### ISSUE THREAD

**Issue by WeiBanjo** [nginxinc/nginx-prometheus-exporter#65]
**Describe the bug**
Nginx exporter fails to export metrics

Exporter was failing on 0.3.0 to export metrics with `path not found` error after upgrade to 0.4.2. Some of the hosts are able to export metrics. But there is one host still has same error.

**Expected behavior**
Export metrics without failure.

**Your environment**
Centos 3.10.0-957.1.3.el7.x86_64

nginx version: nginx/1.15.10 (nginx-plus-r18-p1)
built by gcc 4.8.5 20150623 (Red Hat 4.8.5-36) (GCC)
built with OpenSSL 1.0.2k-fips  26 Jan 2017

nginx exporter version 0.4.2

**Additional context**
```
# Systemd command
nginx-prometheus-exporter -web.listen-address :9113 -nginx.plus -nginx.scrape-uri http://127.0.0.1:8888/api -nginx.ssl-verify false
```

```
# Error log
nginx-prometheus-exporter[13565]: 2019/09/26 21:17:14 Error getting stats: failed to get stats: failed to get stream server zones: expected 200 response, got 404. path=; method=; error.status=404; error.text=path not found; error.code=PathNotFound; request_id=10e67745e5d4466f570cb834aa9c17e0; href=https://nginx.org/en/docs/http/ngx_http_api_module.html
```

**Comment by pleshakov** [comment]
@WeiBanjo thanks for reporting.

This issue is similar to https://github.com/nginxinc/nginx-prometheus-exporter/issues/45

We fixed this problem in Release 0.4.0. Could you possibly double check that the exporter is 0.4.0+ ?

**Comment by WeiBanjo** [comment]
Double checked my host. 0.4.2 fixes my issue.

Thanks.
